src/stdatamodels/jwst/transforms/converters/jwst_models.py

```diff
@@ -20,19 +20,22 @@ class NIRCAMGrismDispersionConverter(TransformConverterBase):
     def from_yaml_tree_transform(self, node, tag, ctx):
         from stdatamodels.jwst.transforms import models
 
-        _fname = getattr(models, node["class_name"])
+        _fname = getattr(models, node["model_type"])
```

Comment:
Additional changes maybe necessary if my suggested changes to the schema are needed. Namely, it may need to branch on single instances vs lists of instances.